test/rdf-star/turtle-star/nt-ttl-star-bad-syntax-1.ttl

```diff
@@ -0,0 +1 @@
+<http://example/a> << <http://example/s> <http://example/p>  <http://example/o> >>  <http://example/z> .
```

Comment:
It seems like you have the same files in [`test/rdf-star/turtle-star/`](https://github.com/RDFLib/rdflib/tree/e0d5746b26d43c9e476b995ee14304173206329f/test/rdf-star/turtle-star) and [`test/turtle-star`](https://github.com/RDFLib/rdflib/tree/e0d5746b26d43c9e476b995ee14304173206329f/test/turtle-star)